File: stuff_policies/src/main/java/android/montreal/devfest/stuff/policies/RandomStuffVerifierImpl.java
package android.montreal.devfest.stuff.policies;

import android.montreal.devfest.stuff.entities.RandomStuffEntity;

class RandomStuffVerifierImpl implements RandomStuffVerifier {
    @Override
    public boolean isValid(RandomStuffEntity stuff) {
        return stuff != null && stuff.text != null && stuff.text.length() != 0;
    }
}

File: stuff_policies/src/main/java/android/montreal/devfest/stuff/policies/RandomStuffVerifierImpl.kt
package android.montreal.devfest.stuff.policies

import android.montreal.devfest.stuff.entities.RandomStuffEntity

internal class RandomStuffVerifierImpl : RandomStuffVerifier {
    override fun isValid(stuff: RandomStuffEntity): Boolean {
        return stuff.text?.length != 0
    }
}
